src/components/Footer/index.tsx

```diff
@@ -0,0 +1,86 @@
+'use client';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import CONFIG from '@/lib/configs';
+import { IconType } from '@/lib/icons';
+import { Flex } from '@/styles/containers';
+import { P } from '@/styles/text';
+import FooterImage from '../../../public/images/footer-image.webp';
+import Icon from '../Icon';
+import * as Styles from './styles';
+
+type SocialLink = {
+  href: string;
+  type: IconType;
+};
+
+const socialLinks: SocialLink[] = [
+  { href: CONFIG.facebook, type: 'facebook' },
+  { href: CONFIG.xTwitter, type: 'x' },
+  { href: CONFIG.linkedin, type: 'linkedin' },
+  { href: CONFIG.instagram, type: 'instagram' },
+];
+
+export default function Footer() {
+  return (
+    <Styles.Footer>
+      <Styles.FooterSection>
+        <Image
+          alt="footer image"
+          src={FooterImage.src}
+          placeholder="blur"
+          blurDataURL={FooterImage.src}
+          quality={100}
+          width={168}
+          height={66}
+        />
```

Comment:
nit: we can pass simply pass in `FooterImage` to `src`, which also tells `Image` about the width and height